shelley/chain-and-ledger/executable-spec/test/Generator/Utxo.hs

```diff
@@ -49,18 +49,22 @@ genTx (LedgerEnv slot _ pparams) (UTxOState utxo _ _ _, dpState) keys = do
   -- output addresses
   recipientAddrs <- genRecipients keys'
 
+  ttl <- Gen.integral $ Range.linear 1 100
+  let slotWithTTL = slot + Slot ttl
```

Comment:
why not use `slot` as minimum for the above `Range.linear` generator?

Reply:
I was copying the style from below, where I chopped the code from.

But it also seems a little more clunky to unpack a Slot, pass the wrapped value, and it `+100` to the generator, then rewrap it into a slot.